### PR TITLE
web/admin: fix misc dual select on different forms

### LINKED ISSUE #11197: Error displaying text
**Describe the bug**
Please see the screenshot below

**To Reproduce**

**Expected behavior**

**Screenshots**
image
image

**Logs**
No Logs to show

**Version and Deployment (please complete the following information):**

-   authentik version: 2024.8.0
-   Deployment: docker-compose

**Additional context**
New installation

### PR DESCRIPTION
**Pull request by BeryJu**
## Details

closes #11197

---

## Checklist

-   [x] Local tests pass (`ak test authentik/`)
-   [x] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -205,7 +205,7 @@ gen: gen-build gen-client-ts
 web-build: web-install  ## Build the Authentik UI
 	cd web && npm run build
 
-web: web-lint-fix web-lint web-check-compile web-test  ## Automatically fix formatting issues in the Authentik UI source code, lint the code, and compile it
+web: web-lint-fix web-lint web-check-compile  ## Automatically fix formatting issues in the Authentik UI source code, lint the code, and compile it
```
[the remark this answers is not in the document]
Comment (BeryJu) [reply Makefile:208]: no this was the usual "unrelated" included thing

**Comment by f0o** [comment]
This didnt solve it fully - Providers are still unable to be created.

Browser Console yields:
```
ak-dual-select.ts:118 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'map')
    at h.willUpdate (ak-dual-select.ts:118:55)
    at h.performUpdate (reactive-element.ts:1439:14)
    at h.scheduleUpdate (reactive-element.ts:1338:25)
    at h._$ET (reactive-element.ts:1310:25)

ak-dual-select.ts:284 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'length')
    at h.render (ak-dual-select.ts:284:40)
    at h.update (lit-element.ts:158:24)
    at h.performUpdate (reactive-element.ts:1441:14)
    at h.scheduleUpdate (reactive-element.ts:1338:25)
    at h._$ET (reactive-element.ts:1310:25)
﻿```

Image used:
```
REPOSITORY                   TAG         IMAGE ID       CREATED        SIZE
ghcr.io/goauthentik/server   2024.8.0    fdc41802befb   2 days ago     1.1GB
```

**Comment by f0o** [comment]
@WasserEsser Do you know when the patch will be feed into a release? I assumed it was cherry picked into the 2024.8 release 2 days ago.

Seems odd not to add this patch to the current release as it defacto breaks the whole functionality of it.

**Pull request merged**